### ISSUE TITLE
HTML rendering as text in incoming webhook

### ISSUE THREAD

**Issue by antgel** [RocketChat/Rocket.Chat#8332]
![rc-webhook-fail](https://user-images.githubusercontent.com/295571/30925583-009f6288-a3bb-11e7-8561-9b242ea9fe06.png)
When I put HTML in the `attachment.text` field, it should be rendered as such. However, it's rendered, tags and all. This was tested with a TFS incoming webhook, which is too long to paste here but looks something like:
```
{
...
"message":{
...
"html":"<a href=\"http://fabrikam-fiber-inc.visualstudio.com/web/wi.aspx?pcguid=74e918bf-3376-436d-bd20-8e8c1287f465&id=5\">Bug #5</a> (Some great new idea!) updated by Jamal Hartnett.",
...
},
...
}
```

The webhook code is similar to the following:
```
class Script {
  process_incoming_request({ request }) {

    return {
      content:{
        "attachments": [{
          "title": request.content.resource.fields["System.Reason"].newValue,
          "title_link": request.content.resource.url,
          "text": request.content.message.html,
        }]
       }
    };
  }
}
```

See the attached image for how it looks in RC.

**Comment by geekgonecrazy** [comment]
Yes this used to work just fine... i'm guessing during some attempt to prevent xss probably broke this.

**Comment by TwizzyDizzy** [comment]
Mhhh... @geekgonecrazy: I think we should first elaborate on whether it is something that is working as expected (=not rendering HTML from hooks) or whether incoming HTML from webhook should be rendered.

I think it shouldn't be rendered. Otherwise you would have to do some sanitizing ... and where would you start with that...

One could also view it differently: if you give away a token hook, you implicitly trust the one using that token and thereby willingly open that "attack vector".

Thoughts? :)

Cheers
Thomas

**Comment by KevLehman** [comment]
This pull request has been inactive for a while, so we’re closing it to keep the backlog manageable.
If you’d like to continue working on it, feel free to reopen or submit a new PR. Thanks for the contribution!
